plugins/backstage-plugin-flux/config.d.ts

```diff
@@ -2,5 +2,6 @@ export interface Config {
   gitops?: {
     /** @visibility frontend */
     baseUrl?: string;
+    readOnly?: boolean;
```

Comment:
I think we need to set the visibility on this attr to frontend too?

Does it work if you

`kubectl apply -k demo`
`./startup.sh`

etc?